compiler/src/main/java/io/neow3j/compiler/converters/ConstantsConverter.java

```diff
@@ -49,4 +61,16 @@ public AbstractInsnNode convert(AbstractInsnNode insn, NeoMethod neoMethod,
         }
         return insn;
     }
+
+    private boolean isDesiredAssertionStatusConst(AbstractInsnNode insn) {
+        if (!insn.getNext().getClass().getCanonicalName().equals(
+                MethodInsnNode.class.getCanonicalName())) {
```

Comment:
For instruction type checks you can use this construct instead of the class comparison:
```insn.getNext().getType() == AbstractInsnNode.METHOD_INSN```